app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt

```diff
@@ -36,12 +33,21 @@ class ClocksActivity : AppCompatActivity(), ClocksView {
     super.onDestroy()
     presenter.unbind()
   }
-  override fun render() {
+
+  override fun render(state: ClockState) {
     // Just to put something in the clocks right now
-    when(currentState) {
-      Stopped -> {}
-      WhiteRunning -> {}
-      BlackRunning -> {}
+    when (state) {
+      Stopped -> { }
```

Comment:
In this state I don't know what to do yet.... Maybe "Stopped" state is unnecessary? At least here. What do you think?

Reply:
I think this state will be a must, since in the stop state we can render the `resume button` or the `settings button`, or wathever we want to render in a pause state :) what do you think?